opm/material/fluidstates/BlackOilFluidState.hpp

```diff
@@ -76,6 +76,18 @@ LhsEval getInvB_(typename std::enable_if<!HasMember_invB<FluidState>::value,
         /FluidSystem::referenceDensity(phaseIdx, pvtRegionIdx);
 }
 
+OPM_GENERATE_HAS_MEMBER(saltConcentration, ) // Creates 'HasMember_saltConcentration<T>'.
+
+template <class FluidState>
+unsigned getSaltConcentration_(typename std::enable_if<HasMember_saltConcentration<FluidState>::value,
```

Comment:
Does salt concentration only come in discrete amounts?  If not, I think the return type should be `double` rather than `unsigned` (`int`).

Reply:
Thanks for noticing.